model.go

```diff
@@ -350,17 +346,17 @@ func generateModelType(f *jen.File, m contentfulModel) {
 	)
 
 	var codes = []jen.Code{}
-	var attrs = map[jen.Code]jen.Code{}
+	var attrs = jen.Dict{}
 	generateModelLinkResolver(m, "items", "includes", "cache")(attrs)
 	for k, v := range attrs {
-		codes = append(codes, jen.Sel(jen.Id("tmp"), jen.Add(k)).Op("=").Add(v).Op(";"))
+		codes = append(codes, jen.Id("tmp").Op(".").Add(k).Op("=").Add(v).Op(";"))
```

Comment:
@nicolai86 this is a rather annoying edge case that I hadn't anticipated... Hmmm